### PR TITLE
Add mkl 11.3.1.150 support

### PR DESCRIPTION
**Pull request by jas02**
Fixes the problem with wrong directory checking. There's no `mkl/bin/intel64` anymore.

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyblocks/i/imkl.py [hunk 1]
```diff
@@ -357,7 +357,7 @@ def sanity_check_step(self):
             if self.cfg['m32']:
                 raise EasyBuildError("Sanity check for 32-bit not implemented yet for IMKL v%s (>= 10.3)", self.version)
             else:
-                mkldirs = ["bin", "mkl/bin", "mkl/bin/intel64", "mkl/lib/intel64", "mkl/include"]
+                mkldirs = ["bin", "mkl/bin", "mkl/lib/intel64", "mkl/include"]
```
Comment (boegel) [inline easybuild/easyblocks/i/imkl.py]: we should only remove the check on `mkl/bin/intel64` for the newest versions of `imkl`, and also check what kind of potential fallout this has (which tools are located in `mkl/bin/intel`, and where are they used by EB)?

**Comment by wpoely86** [comment]
@jas02 please add yourself to the author list. I will pull in your commit.

**Comment by jas02** [comment]
@wpoely86 Done, thanks.

**Comment by boegel** [comment]
will get merged via #756

**Comment by boegel** [comment]
@jas02: can you sync this with develop? it should render this PR empty, since all changes are included in #756, but I want to double check that...

**Comment by jas02** [comment]
Seems to be clean.

**Comment by boegel** [comment]
Just to confirm: all these changes are already in develop, GitHub is not showing the correct diff for some reason...

**Comment by jas02** [comment]
I just manually checked, that changes from #745 are already merged in `develop` branch.

**Pull request closed without merging**